fiftyone/core/threed/splat.py

```
@@ -0,0 +1,115 @@
+            explicitly set to ``"sog"``
+        center_geometry (True): whether to center the splat geometry
+        visible (True): default visibility of the splat in the scene
+        position (None): the position of the splat in object space
+        quaternion (None): the quaternion of the splat in object space
+        scale (None): the scale of the splat in object space
+
+    Raises:
+        ValueError: if the path or format is not supported
+    """
+
+    _asset_path_fields = ["splat_path"]
```

Ruff (0.15.21)

[warning] Mutable default value for class attribute

(RUF012)